src/app/components/get-involved/get-involved.component.spec.ts

```diff
@@ -35,11 +37,19 @@ describe('GetInvolvedComponent', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(GetInvolvedComponent);
+    globalService = TestBed.get(GlobalService);
+    globalServiceSpy = spyOn(globalService, 'scrollToTop');
     component = fixture.componentInstance;
+
+    expect(globalServiceSpy).not.toHaveBeenCalled();
```

Comment:
:smile:

Reply:
hey @lunayach it's nothing wrong to have that but yeah also it doesn't make sense to have that, so will remove that